grammars/puppet.cson

```diff
@@ -394,23 +256,252 @@
       {
         'include': '#single-quoted-string'
       }
+      {
+        'include': '#bareword'
+      }
     ]
-  'variable':
+  'bareword':
```

Comment:
What exactly is a "bareword", and does it have a special meaning in Puppet?  Would `constant.other.puppet` suffice, or do you feel that it's important to distinguish barewords from other constants?